apps/dashboard/app/models/launcher.rb

```diff
@@ -193,9 +193,13 @@ def submit(options)
     render_format = adapter.class.name.split('::').last.downcase
 
     job_script = OodCore::Job::Script.new(**submit_opts(options, render_format))
+    after = Array(options[:after])
+    afterok = Array(options[:afterok])
+    afternotok = Array(options[:afternotok])
+    afterany = Array(options[:afterany])
 
     job_id = Dir.chdir(project_dir) do
-      adapter.submit(job_script)
+      adapter.submit(job_script, after, afterok, afternotok, afterany)
```

Comment:
Can this be put into some sort of helper so this method call can just look like this (I made up the name `after_helper` obviously it should have a better name than that).

```ruby
adapter.submit(job_script, **after_helper(options))
```

Reply:
Makes sense, updated PR.